internal/storage/bucket_handle.go

```diff
@@ -246,7 +245,6 @@ func (b *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsRequ
 			break
 		}
 		if err != nil {
-			err = fmt.Errorf("Error in iterating through objects: %v", err)
```

Comment:
Also, I see this error is given in other methods also like CopyObject. Can we change them also to keep it consistent?